src/subdomains/monitoring/observers/staking-combined.observer.ts

```diff
@@ -39,6 +39,7 @@ export class StakingCombinedObserver extends MetricObserver<StakingData> {
     try {
       data = await this.getStaking();
     } catch (e) {
+      console.error('Exception during monitoring staking combined:', e);
       data = this.getDefaultData();
```

Comment:
I would remove DefaultData completely and stop the process here. This only triggers unwanted monitoring mails.